pkg/users/account-details.js

```diff
@@ -195,16 +192,15 @@
 
     if (!account) {
         return (
-            <EmptyState variant={EmptyStateVariant.small} id="account-failure">
-                <EmptyStateIcon icon={ExclamationCircleIcon} />
-                <Title headingLevel="h1" size="lg">
-                    {_("Account not available or cannot be edited.")}
-                </Title>
-                <EmptyStateSecondaryActions>
-                    <Breadcrumb>
-                        <BreadcrumbItem to="#/">{_("Back to accounts")}</BreadcrumbItem>
-                    </Breadcrumb>
-                </EmptyStateSecondaryActions>
+            <EmptyState variant={EmptyStateVariant.sm} id="account-failure">
+                <EmptyStateHeader titleText={<>{_("Account not available or cannot be edited.")}</>} icon={<EmptyStateIcon icon={ExclamationCircleIcon} />} headingLevel="h1" />
+                <EmptyStateFooter>
+                    <EmptyStateActions>
+                        <Breadcrumb>
+                            <BreadcrumbItem to="#/">{_("Back to accounts")}</BreadcrumbItem>
+                        </Breadcrumb>
+                    </EmptyStateActions>
+                </EmptyStateFooter>
```

Comment:
These 9 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-085951-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)